apps.yml

```diff
@@ -3777,3 +3777,53 @@ apps:
     name: Mattermost IM
     op: auth0
     url: https://auth.mozilla.auth0.com/samlp/dkb2y4Ow9L2PQ6V8bxU7Z1SvNuCSxen6
+- application:
+    authorized_groups:
+    - hris_is_staff
+    - team_moco
+    - team_mofo
+    - mozilliansorg_nda
+    authorized_users: []
+    client_id: EJ9AJ2fn3fIBINxzP62i3t1DZFb1xpfR
+    display: false
+    logo: auth0.png
+    name: Developer Portal (local)
+    op: auth0
+    url: https://developer-portal-127-0-0-1.nip.io/auth/callback
```

Comment:
Heya @jdow - I'm currently blocked on using HTTPS in local development because of a CI issue.

Is it possible to change this to `http://localhost`, at least in the meantime please?